lib/vsc/utils/generaloption.py

```diff
@@ -1705,7 +1706,7 @@ class SimpleOptionParser(ExtOptionParser):
 class SimpleOption(GeneralOption):
     PARSER = SimpleOptionParser
 
-    def __init__(self, go_dict=None, descr=None, short_groupdescr=None, long_groupdescr=None, config_files=None):
+    def __init__(self, go_dict=None, short_groupdescr=None, long_groupdescr=None, config_files=None):
```

Comment:
removing this argument will definitely break a lot of downstream binaries using this.
better `deprecate` it first, like, printing out that it's not used?